Automation/NSG/parameters.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "networkSecurityGroups_Name": {
+            "value": "automation-jb-nsg"
```

Comment:
Make this a parameter, not hard-coded

Reply:
I was under the impression, that the parameters file is meant to be hard coded for easier set up by the user, rather than prompting for a lot of values.

Reply:
You can send in parameters when you create the resource:
`    New-AzResourceGroupDeployment -ResourceGroupName $resourceGroupName -TemplateFile $pathToVNetTemplate -vNetName $vNetName`

Reply:
So for you:
`New-AzResourceGroupDeployment 
    -Name $DeploymentName 
    -ResourceGroupName $ResourceGroupName 
    -TemplateFile $TemplateFilePath 
    -TemplateParameterFile $ParameterFilePath
    -networkSecurityGroups_Name $securityGroupName`